.travis.yml

```diff
@@ -57,11 +57,11 @@ matrix:
   include:
     - os: osx
       env: BUILD_TYPE=Release BUILD_EXTERNAL=0 MAKE_TARGETS="llvm-spirv" MAKE_TEST_TARGET="check-llvm-spirv"
-      osx_image: xcode13
+      osx_image: xcode14
```

Comment:
Do we really need to change version of xcode?

Reply:
Good point, there is not even such a thing as xcode14 it seems.  I'll revert this.